site/gdocs/ProminentLink.tsx

```diff
@@ -14,64 +14,82 @@ export const ProminentLink = (props: {
+        useLinkedDocument(props.url)
+    const { linkedChart, errorMessage: linkedChartErrorMessage } =
+        useLinkedChart(props.url)
+    const errorMessage = linkedDocumentErrorMessage || linkedChartErrorMessage
     const { isPreviewing } = useContext(DocumentContext)
-    if (linkType == "gdoc") {
-        let error: Error | undefined = undefined
-        if (!linkedDocument) {
-            error = {
-                name: "Error in prominent link",
-                message: `Google doc URL ${url} isn't registered. This block will not render when the page is baked.`,
-            }
-        } else if (!linkedDocument.published) {
-            error = {
-                name: "Error in prominent link",
-                message: `Article with slug "${linkedDocument.slug}" isn't published. This block will not render when the page is baked.`,
-            }
-        }
-        if (error) {
-            if (isPreviewing) {
-                return (
-                    <div className={className}>
-                        <BlockErrorFallback
-                            className="span-cols-6"
-                            error={error}
-                        />
-                    </div>
-                )
-            } else return null
-        }
+
+    if (errorMessage) {
+        if (isPreviewing) {
+            return (
+                <div className={props.className}>
+                    <BlockErrorFallback
+                        className="span-cols-6 span-md-cols-10 span-sm-cols-12"
+                        error={{
+                            name: "Error with prominent link",
+                            message: `${errorMessage}\nThis block won't render when the page is published`,
+                        }}
+                    />
+                </div>
+            )
+        } else return null
     }
 
-    // If the link points to a gdoc_post, we can use its metadata
+    // If the link points to a gdoc_post or chart, we can use its metadata
     // But we still allow for overrides written in archie
-    const title = props.title || linkedDocument?.content.title
-    const description = props.description || linkedDocument?.content.excerpt
-    const href = linkedDocument ? `/${linkedDocument.slug}` : url
-    const anchorTagProps = linkedDocument
-        ? {}
-        : { target: "_blank", rel: "noopener noreferrer" }
-    const thumbnail = props.thumbnail || linkedDocument?.content.cover
+    let href: string = props.url
+    let title: string | undefined = props.title
+    let description: string | undefined = props.description
+    let thumbnail: string | undefined = props.thumbnail
+    if (linkType === "gdoc") {
+        href = `/${linkedDocument?.slug}`
+        title = title ?? linkedDocument?.content.title
+        description = description ?? linkedDocument?.content.excerpt
+        thumbnail = thumbnail ?? linkedDocument?.content.cover
```

Comment:
is there a difference between `cover` and `cover-image`? If so `lightningContentProps` should probably be updated

Reply:
Yeah I realized I was going to have to deal with this. Opened a separate PR for it https://github.com/owid/owid-grapher/pull/2106